shumai/cpp/flashlight_binding.cc

```diff
@@ -90,6 +95,34 @@ JSTypedArrayBytesDeallocator genTensorDestroyer() {
   return destroyTensor;
 }
 
+// `grad_in` is Shumai equivalent for Flashlight `gradOutput`
+void* conv2dBackwardData(void* grad_in,
```

Comment:
please prefix with `_` (convention for functions that take in a `Tensor`) and move to the bottom of the file (after `_copy`)

Reply:
Fixed!